src/TBuffer.cpp

```diff
@@ -3171,6 +3175,14 @@ QString TBuffer::bufferToHtml( QPoint P1, QPoint P2 )
     QString fontStyle;
     QString textDecoration;
     bool firstSpan = true;
+    if( !timeBuffer[y].isEmpty() )
+    {
+        firstSpan = false;
```

Comment:
Remember `firstSpan` being `true` initially __is needed__ to ensure that the CSS is present in the first `<span ...>` in the created HTML file and to avoid the insertion of a `\</span>` in the __first__ iteration around the `for(`...`)` that follows here... :worried:

Actually, further examination suggests this is correct after all! :grinning: